### PR TITLE
Read license from file (take 2)

### PR DESCRIPTION
**Pull request by jmccann**
Picking up #1  to be able to read a license template from a file for more versatility.

### CODE REVIEW

**Comment by jmccann** [comment]
I signed it!

**Comment by jmccann** [comment]
@bradrydzewski Not sure if you still cared about #1 but I updated it to work with current master.  Please allow me to use your code.  😄

File: main.go [hunk 1]
```diff
@@ -65,6 +67,26 @@ func main() {
 		Holder: *holder,
 	}
 
+	var t *template.Template
+	if *licensef != "" {
+		d, err := ioutil.ReadFile(*licensef)
+		if err != nil {
+			log.Printf("license file: %s", err)
```
Comment (x1ddos) [inline main.go]:
```suggestion
			log.Printf("license file: %v", err)
```

File: main.go [hunk 2]
```diff
@@ -65,6 +67,26 @@ func main() {
...
+			os.Exit(1)
+		}
+		t, err = template.New("").Parse(string(d))
+		if err != nil {
+			log.Printf("license file: %s", err)
```
Comment (x1ddos) [inline main.go]:
```suggestion
			log.Printf("license file: %v", err)
```

**Comment by x1ddos** [comment]
Thanks @jmccann

**Pull request merged**